itests/org.jboss.tools.hibernate.ui.bot.test/src/org/jboss/tools/hibernate/ui/bot/test/SmokeSuite.java

```diff
@@ -29,18 +29,18 @@
 @RunWith(RedDeerSuite.class)
 @Suite.SuiteClasses({
 
-	CodeGenerationConfigurationTest.class,
-	ConnectionProfileTest.class,
-	ConsoleConfigurationFileTest.class,
-	ConsoleConfigurationTest.class,
-	CreateJPAProjectTest.class,
+//	CodeGenerationConfigurationTest.class, // pass
```

Comment:
As I understand, you want to create smoke suite with only 2 tests. Do you need this commented tests there?

Reply:
Just a mistake. Smoke suite will remain the same.